.gitignore

```diff
@@ -35,3 +35,6 @@ flatpak/bundles
 
 # src/ usually has development version of liblarch
 src/
+
+# Python virtual environments
+venv*/
```

Comment:
Minor nitpick, could we put this in a separate commit?

Reply:
Fair enough - moved to #1064